package.json

```diff
@@ -7,17 +7,20 @@
     "url": "https://github.com/openfisca/openfisca-doc.git"
   },
   "scripts": {
+    "prebuild": "npm run clean",
     "build": "gitbook build",
-    "clean": "rm -rf _book",
+    "clean": "del _book",
+    "deploy": "npm run build && npm run deploy:rsync && git fetch",
```

Comment:
`deploy` is intended for manual use only. The CI calls `deploy:rsync` directly.

I can simplify renaming `deploy:rsync` to `deploy` and letting users run `npm run build && npm run deploy` manually. That should never happen thanks to CI.